openhcl/virt_mshv_vtl/src/processor/mod.rs

```diff
@@ -180,13 +181,15 @@ mod private {
         pub(crate) hv: Option<VtlArray<ProcessorVtlHv, 2>>,
         pub(crate) vp_info: &'a TargetVpInfo,
         pub(crate) runner: &'a mut ProcessorRunner<'b, T::HclBacking>,
-        pub(crate) backing_shared: &'a BackingShared,
     }
 
     pub trait BackingPrivate: 'static + Sized + InspectMut + Sized {
         type HclBacking: hcl::ioctl::Backing;
+        type Shared;
+
+        fn shared(shared: &BackingShared) -> &Self::Shared;
```

Comment:
When is this called? Other than in new below?

Reply:
Yeah, all makes sense, just wanted to make sure this method wasn't being called in multiple places